### PR TITLE
Fix crash when email config missing

### LINKED ISSUE #25: App Crashes When Mailchimp api info missing
### Expected function
Shoudn't crash, should just tell the dev that the values are missing

### Actual function
Crashes

### Steps to reproduce the bug
Start app without a valid mailchimp api key in either config or environment

### PR DESCRIPTION
**Pull request by connorkrupp**
Fixes #25 and #26.

### PR Description

Currently the errors don't work on subscribe, and the app crashes when someone doesn't have the Mailchimp API

### What needs to be reviewed?

Small style changes to the code, mostly

### CODE REVIEW

**Comment by roball24** [comment]
nice fix 👍🏼

File: server/interactors/email.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 
-module.exports.ERRORS = {
+var Errors = {
     ALREADY_SUBSCRIBED: 'ALREADY_SUBSCRIBED',
+    MISSING_CONFIG: 'MISSING_CONFIG',
     UNKNOWN: 'UNKNOWN_ERROR'
 };
 
-module.exports.subscribe = function(email, callback) {
+var subscribe = function(email, callback) {
```
Comment (antoniomika) [inline server/interactors/email.js]: If we're going this route, might as well just use `function subscribe(email, callback)`

File: server/interactors/email.js [hunk 2]
```diff
@@ -20,14 +28,21 @@ module.exports.subscribe = function(email, callback) {
         }
     }, function(err, result) {
         var error = null;
-        if (result.error_count > 0) {
+        console.log("err:", err);
```
Comment (antoniomika) [inline server/interactors/email.js]: Let's use `console.error` for error logging. Also, this should only be done when err isn't null.

File: server/interactors/email.js [hunk 3]
```diff
@@ -20,14 +28,21 @@ module.exports.subscribe = function(email, callback) {
...
+        console.log("res:", result);
```
Comment (antoniomika) [inline server/interactors/email.js]: Do we want to keep this in? If so, maybe we'll want to look into using a debug logger

File: server/interactors/email.js (hunk 2 above)
Comment (connorkrupp) [inline server/interactors/email.js]: That wasn't even supposed to stay in the code haha my b

File: server/interactors/email.js (hunk 3 above)
Comment (connorkrupp) [inline server/interactors/email.js]: I'll just take it out

File: server/interactors/email.js (hunk 1 above)
Comment (connorkrupp) [inline server/interactors/email.js]: Jeah

**Pull request merged**